github/github-release/README.md

```diff
@@ -0,0 +1,97 @@
+# Github Release
+
+It is typical to create a Github tag at the moment of release to introduce a checkpoint in your source code history,
+but in most cases users will need compiled objects or other assets output, not just the raw source code.
+
+Github Releases are a way to track deliverables in your project. Consider them a snapshot in time of the source,
+build output, artifacts, and other metadata associated with a released version of your code.
+
+This `task` can be used to make a github release.
+
+Task can also be used to upload `assets` including `binaries` of the released version, with the release.
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/v1beta1/github/github-release/github-release.yaml
+```
+
+## Parameters
+
+- **TAG_NAME**: The tag where the release will be created from (_e.g:_`v1.0.0`).
+- **NAME**: The Name of the release (_e.g:_`First release`).
+- **BODY**: A short description of the release (_default:_`""`).
+- **OWNER**: The `user name` of the Github owner.
+- **REPO_NAME**: Full name of the `repository` where the release is to made.
+- **BRANCH**: Branch from the repository on which release has to be made (_default:_`master`).
+- **UPLOAD_ASSET_FILE_NAME**: The file name of the asset that needs to be uploaded.
+- **github-token-secret**: The name of the `secret` holding the github-token (_default:_`github-token`).
```

Comment:
I notice that the param names here are a mix of ALL_UPPERCASE and all_lowercase.  Suggest standardizing on a single format.